packages/app/navigation/desktop/ProfileNavigator.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  DrawerContentComponentProps,
+  createDrawerNavigator,
+} from '@react-navigation/drawer';
+import { NavigationState } from '@react-navigation/routers';
+import { View, getVariableValue, useTheme } from '@tamagui/core';
+import * as db from '@tloncorp/shared/db';
+import * as store from '@tloncorp/shared/store';
+import {
+  ContactsScreenView,
+  ScreenHeader,
+  getDisplayName,
+  isWeb,
+} from '@tloncorp/ui';
+import { useCallback } from 'react';
+import { Alert } from 'react-native';
+
+import { AddContactsScreen } from '../../features/contacts/AddContactsScreen';
+import { EditProfileScreen } from '../../features/settings/EditProfileScreen';
+import { UserProfileScreen } from '../../features/top/UserProfileScreen';
+import { ProfileDrawerParamList } from '../types';
+
+const ProfileDrawer = createDrawerNavigator();
+
+function DrawerContent(props: DrawerContentComponentProps) {
+  const state = props.state as NavigationState<ProfileDrawerParamList>;
+  const { navigate } = props.navigation;
+  const focusedRoute = state.routes[props.state.index];
+
+  const { data: userContacts } = store.useUserContacts();
+  const { data: suggestions } = store.useSuggestedContacts();
+
+  const onContactPress = useCallback(
+    (contact: db.Contact) => {
+      navigate('UserProfile', { userId: contact.id });
+    },
+    [navigate]
+  );
+
+  const onContactLongPress = useCallback((contact: db.Contact) => {
+    if (!isWeb && contact.isContactSuggestion) {
+      Alert.alert(`Add ${getDisplayName(contact)}?`, '', [
+        {
+          text: 'Cancel',
+          style: 'cancel',
+        },
+        {
+          text: 'Add Contact',
+          style: 'default',
+          onPress: () => {
+            store.addContact(contact.id);
+          },
+        },
+        {
+          text: 'Decline Suggestion',
+          style: 'destructive',
+          onPress: () => {
+            store.removeContactSuggestion(contact.id);
+          },
+        },
+      ]);
+    }
+  }, []);
+
+  return (
+    <View>
+      <ScreenHeader
+        title="Contacts"
+        rightControls={
+          <ScreenHeader.IconButton
+            type="Add"
+            onPress={() => navigate('AddContacts')}
+          />
+        }
+      />
+      <ContactsScreenView
+        contacts={userContacts ?? []}
+        suggestions={suggestions ?? []}
+        focusedContactId={focusedRoute.params?.userId}
+        onContactPress={onContactPress}
+        onContactLongPress={onContactLongPress}
```

Comment:
Since longpress doesn't work on the web we might want to reveal the menu via a context menu button

Reply:
like right click or you mean a triple dot? I think as is just popping open the profile on the right is probably fine though it would be cool to highlight the "add contact" button.